libz-dev or zlib1g-dev is required for "gem install nokogiri" if nokogiri is a dependance

<!---
1. Verify first that your feature request is not already reported on GitHub.

2. Verify that your feature is not already implemented: https://devilbox.readthedocs.io

3. PLEASE FILL OUT ALL REQUIRED INFORMATION BELOW! Otherwise it might take more time to properly handle this feature request
-->


#### ISSUE TYPE
<!-- DO NOT CHANGE THIS -->
 - Feature request
<!-- DO NOT CHANGE THIS -->


#### SUMMARY
<!-- Explain your feature idea briefly -->
The following error during installation
```
"gcc -o conftest -I/usr/include/x86_64-linux-gnu/ruby-2.3.0 -I/usr/include/ruby-2.3.0/ruby/backward -I/usr/include/ruby-2.3.0 -I. -Wdate-time -D_FORTIFY_SOURCE=2   -g -O2 -fdebug-prefix-map=/build/ruby2.3-TqM27i/ruby2.3-2.3.3=. -fstack-protector-strong -Wformat -Werror=format-security -fPIC  -g -DXP_UNIX -Wall -Wcast-qual -Wwrite-strings -Wmissing-noreturn -Winline conftest.c  -L. -L/usr/lib/x86_64-linux-gnu -L. -Wl,-z,relro -Wl,-z,now -fstack-protector -rdynamic -Wl,-export-dynamic     -lruby-2.3 -lz  -lpthread -lgmp -ldl -lcrypt -lm   -lc "
conftest.c:3:18: fatal error: zlib.h: No such file or directory
 #include <zlib.h>
                  ^
compilation terminated.
checked program was:
/* begin */
 1: #include "ruby.h"
 2: 
 3: #include <zlib.h>
 4: 
 5: /*top*/
 6: extern int t(void);
 7: int main(int argc, char **argv)
 8: {
 9:   if (argc > 1000000) {
10:     printf("%p", &t);
11:   }
12: 
13:   return 0;
14: }
15: extern void gzdopen();
16: int t(void) { gzdopen(); return 0; }
/* end */
```

#### Goal
<!-- Explain what you actually want to accomplish with that feature -->
Support ruby with essential libs.

--- a/.devilbox/www/config.php
+++ b/.devilbox/www/config.php
@@ -14,7 +14,7 @@ putenv('RES_OPTIONS=retrans:1 retry:1 timeout:1 attempts:1');
 
 
 $DEVILBOX_VERSION = 'v1.0.1';
-$DEVILBOX_DATE = '2019-04-27';
+$DEVILBOX_DATE = '2019-05-21';
 $DEVILBOX_API_PAGE = 'devilbox-api/status.json';
 
 //

--- a/.devilbox/www/config.php
+++ b/.devilbox/www/config.php
@@ -13,7 +13,7 @@ error_reporting(-1);
 putenv('RES_OPTIONS=retrans:1 retry:1 timeout:1 attempts:1');
 
 
-$DEVILBOX_VERSION = 'v1.0.1';
+$DEVILBOX_VERSION = 'v1.0.2';
 $DEVILBOX_DATE = '2019-05-21';
 $DEVILBOX_API_PAGE = 'devilbox-api/status.json';
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,7 +5,7 @@ major versions.
 
 
 
-## [Unreleased]
+## Bugfix Release v1.0.2 (2019-05-21)
 
 #### Fixed
 - Fixed various typos in documentation

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,13 +11,20 @@ major versions.
 - Fixed various typos in documentation
 - Fix CI tests: They still expected a mounted mail directory instead of a Docker volume
 - Fix mods for PHP 8.0
+- Make npm binaries available in $PATH
 - Remove orphaned mentions of HOST_PATH_MYSQL_DATADIR
+- Allow symlinks in autostart scripts
 
 #### Added
 - Documentation: Setup ProcessWire
 - Tool: Angular CLI
 - Tool: Laravel Lumen
+- Tool: [prestissimo](https://github.com/hirak/prestissimo)
+- Tool: [yq](https://github.com/mikefarah/yq)
+- Module: OAuth
 
+#### Removed
+- Enchant module for PHP 7.4 and PHP 8.0 (build breaks)
 
 
 ## Bugfix Release v1.0.1 (2019-03-24)

--- a/README.md
+++ b/README.md
@@ -48,31 +48,35 @@ Furthermore, the Devilbox provides an **identical** and **reproducible developme
 * [Docker Engine 1.12.0+](https://docs.docker.com/compose/compose-file/compose-versioning/#version-21)
 * [Docker Compose 1.9.0+](https://docs.docker.com/compose/compose-file/compose-versioning/#version-21)
 
-**Important**
-
-Before updating the Devilbox ensure to always check for breaking changes until the stable release is out:
-
-1. [Forum: Announcements](https://devilbox.discourse.group/c/announcements)
-2. [Changelog](CHANGELOG.md)
-3. [Updating](UPDATING.md)
 
 ## Architecture
+
+#### Available Stacks
 
 The Devilbox aims to be a swiss army knife for local development by providing you all the services
 you would ever need. To get an idea about the architecture behind it and to also see what's available
 have a look at the following diagrams and tables.
 
-#### Smallest Stack
-This is the smallest possible and fully functional stack you can run:
-
-<img width="300" style="width:300px" title="Devilbox stack" src="https://raw.githubusercontent.com/devilbox/artwork/master/submissions_diagrams/cytopia/02/png/architecture-small.png" />
-
-> [Devilbox artwork](https://github.com/devilbox/artwork)
-
-#### Full Stack
-To better understand what is actually possible have a look at the full example:
-
-<img title="Devilbox stack" src="https://raw.githubusercontent.com/devilbox/artwork/master/submissions_diagrams/cytopia/01/png/architecture-full.png" />
+<table width="100%" style="width:100%;display:table">
+ <tr>
+  <td width="30%" style="width:30%; vertical-align:top;">
+   <h4>Smallest stack</h4>
+   <p>This is the smallest possible and fully functional stack you can run</p>
+  </td>
+  <td width="70%" style="width:70%; vertical-align:top;">
+   <h4>Full stack</h4>
+   <p>To better understand what is actually possible have a look at the full example</p>
+  </td>
+ </tr>
+ <tr>
+  <td>
+   <img width="300" style="width:300px" title="Devilbox stack" src="https://raw.githubusercontent.com/devilbox/artwork/master/submissions_diagrams/cytopia/02/png/architecture-small.png" />
+  </td>
+  <td>
+   <img title="Devilbox stack" src="https://raw.githubusercontent.com/devilbox/artwork/master/submissions_diagrams/cytopia/01/png/architecture-full.png" />
+  </td>
+ </tr>
+</table>
 
 > [Devilbox artwork](https://github.com/devilbox/artwork)
 
@@ -106,9 +110,10 @@ simplify your every-day life. If you ever run into any unforseen issues, feel fr
 <table width="100%" style="width:100%; display:table;">
  <thead>
   <tr>
-   <th width="33%" style="width:33%;"><h3><a target="_blank" href="https://devilbox.readthedocs.io">Documentation</a></h3></th>
-   <th width="33%" style="width:33%;"><h3><a target="_blank" href="https://gitter.im/devilbox/Lobby">Chat</a></h3></th>
-   <th width="33%" style="width:33%;"><h3><a target="_blank" href="https://devilbox.discourse.group">Forum</a></h3></th>
+   <th width="25%" style="width:25%;"><h3><a target="_blank" href="https://devilbox.readthedocs.io">Documentation</a></h3></th>
+   <th width="25%" style="width:25%;"><h3><a target="_blank" href="https://gitter.im/devilbox/Lobby">Chat</a></h3></th>
+   <th width="25%" style="width:25%;"><h3><a target="_blank" href="https://devilbox.discourse.group">Forum</a></h3></th>
+   <th width="25%" style="width:25%;"><h3><a target="_blank" href="https://github.com/devilbox/flames">Flames</a></h3></th>
   </tr>
  </thead>
  <tbody style="vertical-align: middle; text-align: center;">
@@ -128,11 +133,17 @@ simplify your every-day life. If you ever run into any unforseen issues, feel fr
      <img title="Devilbox Forums" name="Forum" src="https://raw.githubusercontent.com/cytopia/icons/master/400x400/discourse.png" />
     </a>
    </td>
+   <td>
+    <a target="_blank" href="https://github.com/devilbox/flames">
+     <img title="Devilbox Flames" name="Flames" src="https://raw.githubusercontent.com/cytopia/icons/master/400x400/flames2.png" />
+    </a>
+   </td>
   </tr>
   <tr>
   <td><a target="_blank" href="https://devilbox.readthedocs.io">devilbox.readthedocs.io</a></td>
   <td><a target="_blank" href="https://gitter.im/devilbox/Lobby">gitter.im/devilbox</a></td>
   <td><a target="_blank" href="https://devilbox.discourse.group">devilbox.discourse.group</a></td>
+  <td><a target="_blank" href="https://github.com/devilbox/flames">github.com/devilbox/flames</a></td>
   </tr>
  </tbody>
 </table>
@@ -151,15 +162,15 @@ simplify your every-day life. If you ever run into any unforseen issues, feel fr
  <tbody style="vertical-align: bottom;">
   <tr>
    <td>
-    <pre># Get the Devilbox
-$ git clone https://github.com/cytopia/devilbox</pre>
-<pre># Create docker-compose environment file
-$ cd devilbox
-$ cp env-example .env</pre>
-<pre># Edit your configuration
-$ vim .env</pre>
-<pre># Start all container
-$ docker-compose up</pre>
+<div class="highlight highlight-source-shell"><pre># Get the Devilbox
+git clone https://github.com/cytopia/devilbox</pre></div>
+<div class="highlight highlight-source-shell"><pre># Create docker-compose environment file
+cd devilbox
+cp env-example .env</pre></div>
+<div class="highlight highlight-source-shell"><pre># Edit your configuration
+vim .env</pre></div>
+<div class="highlight highlight-source-shell"><pre># Start all container
+docker-compose up</pre></div>
    </td>
    <td>
     1. Clone <code>https://github.com/cytopia/devilbox</code> to <code>C:\devilbox</code> with <a href="https://git-scm.com/downloads">Git for Windows</a><br/><br/>
@@ -183,7 +194,7 @@ C:\devilbox> docker-compose up</pre></div>
 The above will start all containers, you can however also just start the containers you actually need. This is achieved by simply specifying them in the docker-compose command.
 
 ```bash
-$ docker-compose up httpd php mysql redis
+docker-compose up httpd php mysql redis
 ```
 > **Documentation:**
 > [Start only some container](https://devilbox.readthedocs.io/en/latest/getting-started/start-the-devilbox.html#start-some-container)
@@ -345,10 +356,9 @@ Every single attachable container comes with many different versions. In order t
   </tbody>
 </table>
 
-<strong><sup>[1]</sup></strong> <strong>PHP 5.2</strong> is available to use, but it is not officially supported. The Devilbox intranet does not work with this version as PHP 5.2 does not support namespaces.
-Furthermore PHP 5.2 does only work with Apache 2.4, Nginx stable and Nginx mainline. It does not work with Apache 2.2. Use at your own risk.
+<small><strong><sup>[1]</sup></strong> <strong>PHP 5.2</strong> is available to use, but it is not officially supported. The Devilbox intranet does not work with this version as PHP 5.2 does not support namespaces. Furthermore PHP 5.2 does only work with Apache 2.4, Nginx stable and Nginx mainline. It does not work with Apache 2.2. Use at your own risk.</small>
 
-<strong><sup>[2]</sup></strong> <strong>PHP 7.4</strong> and <strong>PHP 8.0</strong> are upcoming unreleased versions of PHP, which are directly built out of their [official git branches](https://github.com/php/php-src/) every night to assure you will leverage their latest features.
+<small><strong><sup>[2]</sup></strong> <strong>PHP 7.4</strong> and <strong>PHP 8.0</strong> are upcoming unreleased versions of PHP, which are directly built out of their [official git branches](https://github.com/php/php-src/) every night to assure you will leverage their latest features.</small>
 
 > **Documentation:**
 > [Change container versions](https://devilbox.readthedocs.io/en/latest/getting-started/change-container-versions.html)
@@ -433,12 +443,12 @@ Navigate the the Devilbox directory and type the below listed command:
  <tbody style="vertical-align: bottom;">
   <tr>
    <td>
-    <pre>host> ./shell.sh
-devilbox@php-7.0.19 in /shared/httpd $</pre>
+<div class="highlight highlight-source-shell"><pre>host> ./shell.sh
+devilbox@php-7.0.19 in /shared/httpd $</pre></div>
    </td>
    <td>
-    <pre>C:\devilbox> shell.bat
-devilbox@php-7.0.19 in /shared/httpd $</pre>
+<div class="highlight highlight-source-shell"><pre>C:\devilbox> shell.bat
+devilbox@php-7.0.19 in /shared/httpd $</pre></div>
    </td>
   </tr>
  </tbody>
@@ -525,6 +535,10 @@ The Devilbox has everything setup for you. The only thing you will have to insta
   <tr>
     <td>:star: Xdebug</td>
     <td>Xdebug and a full blown PHP-FPM server is ready to serve.</td>
+  </tr>
+  <tr>
+    <td>:star: Devilbox Flames</td>
+    <td>Devilbox community plugins a.k.a. Devilbox Flames.</td>
   </tr>
   <tr>
     <td>:star: Many more</td>

--- a/README.md
+++ b/README.md
@@ -48,7 +48,6 @@ Furthermore, the Devilbox provides an **identical** and **reproducible developme
 * [Docker Engine 1.12.0+](https://docs.docker.com/compose/compose-file/compose-versioning/#version-21)
 * [Docker Compose 1.9.0+](https://docs.docker.com/compose/compose-file/compose-versioning/#version-21)
 
-
 ## Architecture
 
 #### Available Stacks
@@ -714,7 +713,7 @@ The Devilbox is a development stack, so it is made sure that a lot of PHP module
 | date           |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |
 | dba            |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |
 | dom            |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |
-| enchant        |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |
+| enchant        |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |         |         |
 | ereg           |         |    ✔    |    ✔    |    ✔    |    ✔    |         |         |         |         |         |         |
 | exif           |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |
 | FFI            |         |         |         |         |         |         |         |         |         |    🗸    |    🗸    |
@@ -746,6 +745,7 @@ The Devilbox is a development stack, so it is made sure that a lot of PHP module
 | mysql          |    ✔    |    ✔    |    🗸    |    🗸    |    🗸    |         |         |         |         |         |         |
 | mysqli         |    ✔    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |
 | mysqlnd        |         |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |
+| OAuth          |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |         |         |
 | oci8           |         |    d    |    d    |    d    |    d    |    d    |    d    |    d    |    d    |    d    |    d    |
 | openssl        |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |    ✔    |
 | pcntl          |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |    🗸    |

--- a/README.md
+++ b/README.md
@@ -58,11 +58,11 @@ have a look at the following diagrams and tables.
 
 <table width="100%" style="width:100%;display:table">
  <tr>
-  <td width="30%" style="width:30%; vertical-align:top;">
+  <td width="30%" valign="top" style="width:30%; vertical-align:top;">
    <h4>Smallest stack</h4>
    <p>This is the smallest possible and fully functional stack you can run</p>
   </td>
-  <td width="70%" style="width:70%; vertical-align:top;">
+  <td width="70%" valign="top" style="width:70%; vertical-align:top;">
    <h4>Full stack</h4>
    <p>To better understand what is actually possible have a look at the full example</p>
   </td>

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -95,7 +95,7 @@ services:
   # PHP
   # ------------------------------------------------------------
   php:
-    image: devilbox/php-fpm:${PHP_SERVER}-work-0.87
+    image: devilbox/php-fpm:${PHP_SERVER}-work-0.88
     hostname: php
 
     ##

--- a/docs/_includes/links/tools.rst
+++ b/docs/_includes/links/tools.rst
@@ -273,6 +273,12 @@
      Yarn <img src="https://raw.githubusercontent.com/cytopia/icons/master/11x11/ext-link.png" />
    </a>
 
+.. |ext_lnk_tool_yq| raw:: html
+
+   <a target="_blank" href="https://github.com/mikefarah/yq">
+     yq <img src="https://raw.githubusercontent.com/cytopia/icons/master/11x11/ext-link.png" />
+   </a>
+
 .. |ext_lnk_tool_github_issues| raw:: html
 
    <a target="_blank" href="https://github.com/cytopia/devilbox/issues">

--- a/docs/readings/available-tools.rst
+++ b/docs/readings/available-tools.rst
@@ -105,6 +105,8 @@ The PHP container is your workhorse and these are your tools:
 +----------------------+---------------------------------------+
 | ``yarn``             | |ext_lnk_tool_yarn|                   |
 +----------------------+---------------------------------------+
+| ``yq``               | |ext_lnk_tool_yq|                     |
++----------------------+---------------------------------------+
 
 .. note::
    If you are in need of other tools, open up an issue at |ext_lnk_tool_github_issues|

